docs/asm-gke-terraform/README.md

```diff
@@ -14,39 +14,40 @@ The following services will be required for this:
 ## Quickstart with Terraform
 In your Cloud Shell, follow the steps outlined:
 
-### 1.  Clone this repo and cd to this directory
+### 1. Clone this repo and cd to this directory
 ```
 git clone https://github.com/GoogleCloudPlatform/anthos-service-mesh-samples
 cd anthos-service-mesh-samples/docs/asm-gke-terraform
 ```
-### 2. Set your Google Cloud Platform `PROJECT_ID`
+### 2. Configure the project to use
 
-Set the `PROJECT_ID` environment variable:
-
-To enable the above services, run the following in your terminal
+Terraform will use the `GOOGLE_CLOUD_PROJECT` environment variable to set the default project to manage resources in.
 ```
-export PROJECT_ID="<YOUR_PROJECT_ID>"
-gcloud config set project $PROJECT_ID
+gcloud config set project <YOUR_PROJECT_ID>
+GOOGLE_CLOUD_PROJECT="$(gcloud config get-value project)"
+export GOOGLE_CLOUD_PROJECT
 ```
-### 3.  Create a `variables.tfvars` to set your project ID for Terraform
-Run the following command to create a `variables.tfvars` file
+
+### 3. Enable the Cloud Resource Manager API
+
 ```
-echo "project_id = \"$PROJECT_ID\"" > terraform.tfvars
+gcloud services enable cloudresourcemanager.googleapis.com
 ```
+The google_project Terraform datasource uses this API to read project metadata.
```

Comment:
**Praise:** Thanks for mentioning this tip about `cloudresourcemanager.googleapis.com`.